### ISSUE TITLE
Feature Request: Add comment/commit message to `helm install`/`helm upgrade`

### ISSUE THREAD

**Issue by huang-jy** [helm/helm#6413]
Output of `helm version`:
```
Client: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.14.1", GitCommit:"5270352a09c7e8b6e8c9593002a73535276507c0", GitTreeState:"clean"}
```

Output of `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.9-2+4a03651a7e7e04", GitCommit:"4a03651a7e7e04a0021b2ef087963dfb7bd0a17e", GitTreeState:"clean", BuildDate:"2019-08-16T18:57:20Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.6-gke.13", GitCommit:"fcbc1d20b6bca1936c0317743055ac75aef608ce", GitTreeState:"clean", BuildDate:"2019-06-19T20:50:07Z", GoVersion:"go1.11.5b4", Compiler:"gc", Platform:"linux/amd64"}
```

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
GKE

After doing an install or upgrade, there is no option to provide a message to add to the history, so when doing `helm history {release}` we get something like this:

```
REVISION	UPDATED                 	STATUS    	CHART               	DESCRIPTION     
1       	Tue Sep 10 07:42:28 2019	SUPERSEDED	chart1.0.0	          Install complete
2       	Tue Sep 10 08:09:18 2019	SUPERSEDED	chart1.0.0	          Upgrade complete
3       	Tue Sep 10 08:32:34 2019	SUPERSEDED	chart1.0.0	          Upgrade complete
4       	Wed Sep 11 10:23:42 2019	SUPERSEDED	chart1.0.0	          Upgrade complete
5       	Wed Sep 11 10:31:22 2019	SUPERSEDED	chart1.0.0	          Upgrade complete
6       	Wed Sep 11 10:55:31 2019	SUPERSEDED	chart1.0.0	          Upgrade complete
7       	Wed Sep 11 11:08:41 2019	SUPERSEDED	chart1.0.0	          Upgrade complete
8       	Wed Sep 11 14:21:24 2019	DEPLOYED  	chart1.0.0	          Upgrade complete
```

But from this, I have no idea who what changes were bundled into, say the revision 5 upgrade from revision 4.

Is there a way we can add an option to add a message to the upgrade/install (upgrade more than install) so for example:

```
REVISION	UPDATED                 	STATUS    	CHART                 	DESCRIPTION        MESSAGE
1       	Tue Sep 10 07:42:28 2019	SUPERSEDED	chart1.0.0	          Install complete  
2       	Tue Sep 10 08:09:18 2019	SUPERSEDED	chart1.0.0	          Upgrade complete  upgrade to v1
3       	Tue Sep 10 08:32:34 2019	SUPERSEDED	chart1.0.0	          Upgrade complete  add additional features
4       	Wed Sep 11 10:23:42 2019	SUPERSEDED	chart1.0.0	          Upgrade complete  blah
5       	Wed Sep 11 10:31:22 2019	SUPERSEDED	chart1.0.0	          Upgrade complete  blah2
6       	Wed Sep 11 10:55:31 2019	SUPERSEDED	chart1.0.0	          Upgrade complete  blah blah
7       	Wed Sep 11 11:08:41 2019	SUPERSEDED	chart1.0.0	          Upgrade complete  blah blah blah
8       	Wed Sep 11 14:21:24 2019	DEPLOYED  	chart1.0.0	          Upgrade complete  blah blah blah blah
```

This could also be used to put something like "upgrade triggered by {username}"

**Comment by bacongobbler** [comment]
`helm install --description`?

https://github.com/helm/helm/blob/master/cmd/helm/install.go#L228

**Comment by huang-jy** [comment]
Oh, not sure how I missed that ^^

Thanks :)

**Comment by aoz-turk** [comment]
Hey there,

Description option is not available in helm v3.0.0. I think it is a useful option.
